smc.c

```diff
@@ -182,15 +203,30 @@ int main(int argc, char *argv[])
       }
     }
 
+    int nfans = 0;
+    double fans[10];
+
     SMCOpen();
     double temperature = SMCGetTemperature(SMC_KEY_CPU_TEMP);
+    for (int i = 0; i < 10; i++) {
+	    char key[5] = SMC_KEY_FAN0_RPM_CUR;
+	    key[1] += i;
+	    double speed = SMCGetFanSpeed(key);
+	    if (speed != 0) {
+		    fans[nfans++] = speed;
+	    }
+    }
     SMCClose();
 
     if (scale == 'F') {
       temperature = convertToFahrenheit(temperature);
     }
 
-    printf("%0.1f°%c\n", temperature, scale);
+    printf("%0.1f°%c", temperature, scale);
+    for (int i = 0; i < nfans; i++) {
```

Comment:
Can you add a `getopt` option so it only prints the fans if an option is passed? Otherwise, it may break existing scripts.